packages/foundry.admin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@osdk/foundry.admin",
-  "version": "3.0.0-beta.1",
```

Comment:
Restoring version back to 2.0.0 so we don't ship this as 3.0.0 when we version next